models/chaincodes.js

```diff
@@ -0,0 +1,15 @@
+const firestore = require('../utils/firestore')
+const admin = require('firebase-admin')
+
+const chaincodeModel = firestore.collection('chaincodes')
+const storeChaincode = async (username) => {
+  const userchaincode = await chaincodeModel.add({
+    username,
+    timestamp: admin.firestore.Timestamp.fromDate(new Date())
+  })
+  return userchaincode.id

```

Comment:
Let's put this code in `try - catch` with an apt error message for easy debugging

Reply:
Also any reasons to not use `now` method for timestamp?

Reply:
now() method doesn't give timestamp in UTC format, so used this instead.